libpod/lock/shm/shm_lock.go

```diff
@@ -0,0 +1,188 @@
+package shm
+
+// #cgo LDFLAGS: -lrt -lpthread
+// #include "shm_lock.h"
+// const uint32_t bitmap_size_c = BITMAP_SIZE;
+import "C"
+
+import (
+	"syscall"
+
+	"github.com/pkg/errors"
+)
+
+var (
+	bitmapSize uint32 = uint32(C.bitmap_size_c)
+)
+
+// SHMLocks is a struct enabling POSIX semaphore locking in a shared memory
+// segment.
+type SHMLocks struct { // nolint
```

Comment:
Could you add a comment saying what is the lit warning being silenced, please, so that it’s noticeable if _some other_ lint warning ever appears?

Reply:
Never mind, the commit already says what this is about.